lib/graphql/preload/loader.rb

```diff
@@ -34,7 +34,7 @@ def perform(records)
       end
 
       private def preload_association(records)
-        if ActiveRecord::VERSION::MAJOR > 3
+        if ActiveRecord.version >= Gem::Version.new('4.1.0.beta1')
```

Comment:
`ActiveRecord#version` is not available in Rails 3. Is there anything wrong with using `ActiveRecord::VERSION`?

Reply:
I'll fix this tomorrow. Just tried another fix, but that will work only on Ruby >= 2. I now think that we need to support >= 1.8.7 due to minimum AR version being 3.2 for the gem.